roles/ood_user_reg_cloud/templates/service.j2

```diff
@@ -3,7 +3,7 @@ Description=uWSGI server for flask user registration
 After=network.target
 
 [Service]
-User={{ RegUser_app_user }}
+User=apache
```

Comment:
Could you ensure your base is up-to-date with feat-openstack.  There seems to be a `Type=simple` that is missing from your [Service] block.